appserver/extras/docker-images/pom.xml

```diff
@@ -91,7 +119,25 @@
                                         </tags>
                                         <cleanup>none</cleanup>
                                         <noCache>${docker.noCache}</noCache>
-                                        <dockerFile>Dockerfile</dockerFile>
+                                        <dockerFile>${project.build.directory}/antrun/Dockerfile.jdk8</dockerFile>
+                                        <assembly>
+                                            <mode>tar</mode>
+                                            <descriptor>assembly.xml</descriptor>
+                                            <tarLongFileMode>gnu</tarLongFileMode>
+                                        </assembly>
+                                    </build>
+                                </image>
+                                <image>
+                                    <name>${docker.payara.repository}</name>
+                                    <build>
+                                        <!-- On Windows with default setting ${*}, $PATH would get filtered as well -->
+                                        <filter>@</filter>
+                                        <tags>
+                                            <tag>${docker.payara.tag}-jdk11</tag>
```

Comment:
There's no -jdk8 on the jdk8 image tag. perhaps it should be introduced for consistency?